Buildings/Resources/Scripts/Dymola/Experimental/DHC/Loads/Cooling/Examples/BuildingTimeSeriesWithETS.mos

```diff
@@ -0,0 +1,38 @@
+// Simulate
+simulateModel("Buildings.Experimental.DHC.Loads.Cooling.Examples.BuildingTimeSeriesWithETS", tolerance=1e-6, stopTime=31536000, method="Dassl", numberOfIntervals=500, resultFile="BuildingTimeSeriesWithETS");
+// Plot commands
+removePlots(true);
+createPlot(id=1, 
+  position={11, 19, 713, 564}, 
+  y={"buiWitETS.bui.loa.y[1]", "buiWitETS.ets.Q_flow"}, 
+  range={-30000, 0}, 
+  grid=true, 
+  filename="BuildingTimeSeriesWithETS.mat",
+  colors={{28,108,200}, {238,46,47}},  
+  timeUnit="d", 
+  displayUnits={"W", "W"}); 
+createPlot(id=1, 
+  position={11, 19, 713, 280}, 
+  y={"senMasFlo.m_flow", "buiWitETS.ets.senMasFlo.m_flow"}, 
+  range={0.0,0.5,0.1,0.1}, 
+  grid=true, 
+  subPlot=2, 
+  colors={{28,108,200}, {238,46,47}}, 
+  timeUnit="d");
+createPlot(id=2, 
+  position={734, 19, 720, 561}, 
+  y={"supChiWat.T", "buiWitETS.ets.senTDisSup.T", "buiWitETS.ets.senTBuiSup.T"}, 

```

Comment:
- [ ] Remove duplicate variables
- [ ] Have supply and return temperatures plotted in the same frame